dask/dataframe/tests/test_dataframe.py

```diff
@@ -172,6 +172,20 @@ def test_Index():
         pytest.raises(AttributeError, lambda: ddf.index.index)
 
 
+def test_axes():
+    pdf = pd.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    df = dd.from_pandas(pdf, npartitions=2)
+    assert len(df) == 2
+    assert all(assert_eq(d, p) for d, p in zip(df.axes, pdf.axes))
+
+
+def test_series_axes():
+    ps = pd.Series(["abcde"])
+    ds = dd.from_pandas(ps, npartitions=2)
+    assert len(ds.axes) == 1
```

Comment:
Let's compare to the expected `len` from pandas instead. What you already have seems right, but comparing to pandas directly will be more clear to readers and robust to potential future changes

```suggestion
    assert len(ds.axes) == len(ps.axes)
```

Reply:
Good suggestion :+1: 
Adjusted in https://github.com/dask/dask/pull/8069/commits/6dd6fafad932c4d5383afa60cfe6dee324123668